osbs/api.py

```diff
@@ -1024,3 +1025,13 @@ def delete_config_map(self, name):
         """
         response = self.os.delete_config_map(name)
         return response
+
+    @contextmanager
+    def retries_disabled(self):
+        """
+        Context manager to disable retries on requests
+        :returns: OSBS object
+        """
+        self.os.retries_enabled = False
+        yield self
```

Comment:
No need to actually yield a value btw. Just `yield` will do the trick. Thinking from a usage point of view, you probably want this:

```python
def my_only_once_method(self):
  with self.retries_disabled():
    self.os.something(...)
```

and not this:

```python
def my_only_once_method(self):
  with self.retries_disabled() as disabled:
    disabled.os.something(...)
```